lib/tasks/data_migration.rake

```diff
@@ -76,4 +76,15 @@ namespace :data_migration do
       puts "Updated #{args[:key]} in #{list.title} to #{new_criteria}"
     end
   end
+
+  desc "Fetch descriptions for single page subscription lists which don't have descriptions"
+  task fetch_subscriber_list_descriptions: :environment do |_t, _args|
+    SubscriberList.where.not(content_id: nil).where(description: nil).find_each do |list|
+      new_description = Services.content_store.content_item(list.url).to_h["description"]
+      list.description = new_description
+      list.save!
```

Comment:
This could just be `list.update!(description: new_description)`

Reply:
Thanks, I'm still not used to `?` and `!` in method names